toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -79,7 +79,7 @@ while read -r package || [ -n "$package" ]; do
     install_one_toolchain_rpm "$package"
 done < "$packages"
 
-# If the host machine rpm version is >= 4.16 (such as Mariner 2.0), it will create an "sqlite" rpm database backend incompatible with Mariner 1.0 (which uses "bdb")
+# If the host machine rpm version is >= 4.16 (such as Azure Linux 2.0), it will create an "sqlite" rpm database backend incompatible with Azure Linux 1.0 (which uses "bdb")
```

Comment:
restore this, and then add Azure Linux 3.0

such as Mariner 2.0 or Azure Linux 3.0

Reply:
Updated